prime-router/docs/onboarding-users/receiver-onboarding/receiver-testing.md

```diff
@@ -0,0 +1,293 @@
+# How to test receiver transforms,settings and filters
+
+Anytime a receiver transform, setting or filter is created or changed, it needs to be appropriately tested to ensure 
+that the change behaves as expected and no unintended changes are introduced. Since the transforms, settings and filters
+all work together to create the final message sent to a receiver, integration testing where a report is submitted to the
+ReportStream application and the final product as it would be sent to the receiver is compared to expected should be
+performed on any change.
+
+## Testing 
+
+### Testing Transforms
+Basic testing on transforms can be performed using the fhirdata CLI command locally:
+
+Example:
+```
+./prime fhirdata -i "@/c/user/some-file-location.fhir" -o "@/c/user/some-file-location.fhir" output-file.fhir -s "classpath:/metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform.yml" --output-format FHIR
+
+Usage: prime fhirdata [<options>]
+
+  Process data into/from FHIR
+
+Options:
+  -i, --input-file=<path>       Input file to process
+  -o, --output-file=<path>      output file
+  --output-format=(HL7|FHIR)    output format
+  --enrichment-schemas=<text>   comma separated enrichment schema name(s) from
+                                current directory
+  --diff-hl7-output=<text>      when true, diff the the input HL7 with the
+                                output, can only be used going HL7 -> FHIR ->
+                                HL7
+  -r, --receiver-schema=<text>  Receiver schema location. Required for HL7
+                                output.
+  -s, --sender-schema=<text>    Sender schema location
+  --input-schema=<text>         Mapping schema for input file
+  -h, --help                    Show this message and exit
+
+```
```

Comment:
Maybe we can update this, now that we're able to pass a receiver name and org to the fhirdata command:
```
  --receiver-name=<text>         Name of the receiver settings to use
  --org=<text>                   Name of the org settings to use
  --receiver-setting-env=<text>  Environment that specifies where to get the receiver settings
```
The command will now show if there are any errors with the filters or if the report got filtered out.